notebooks_tsqr/NightLog.ipynb

```diff
@@ -0,0 +1,271 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Paste contents of source_adapters.py in new cell below this one.
</pre><p>Are you planning on having a different PR to implement that so it works in Times Square, or should we do this now? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_logging_and_reporting/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Argg,  it should be done now! Thanks!!